js/shopping_cart.js

```diff
@@ -33,10 +34,13 @@ function calculateTotal() {
   document.getElementById("total").innerHTML = currentTotal;
 }
 
+function addToTable() {}
+
 if (document.getElementById("addToCart")) {
```

Comment:
It appears this code needs to be executed no matter what so I would put this above all of your other functions.

Reply:
I'm talking about the if(document ... ) part of course

Reply:
so for some reason it doesn't work if you don't have that if(document) line of code. I stole that from a stackoverflow answer and it solved my problem so I didn't question it